Tasks/NpmAuthenticate/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,42 +1,28 @@
 {
   "loc.friendlyName": "npm Authenticate",
-  "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613746)",
-  "loc.description": "Authenticate a npmrc for tasks which use npm.",
+  "loc.helpMarkDown": "",
```

Comment:
Talk to Alex regarding the docs for this new task, he should be able to give you a new fwlink.